vignettes/download_github_comments.Rmd

```diff
@@ -27,40 +36,41 @@ require(magrittr)
 require(gt)
 ```
 
+## Project Configuration File
```

Comment:
Please show a block that shows the user which code block they need to find and change if they need to change any parameters. Maybe even think about explaining some of the more important variables so that users don't have to jump back and forth to find what the variable does, where it's initialized, etc.